src/Application/Components/MapCanvas/MapCanvas.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { fabric } from 'fabric-webpack';
+
+/**
+ * No Pin functionality yet.
+ */
+const MapCanvas = React.createClass({
+  propTypes: {
+    pins: React.PropTypes.object,
+  },
+
+  setMap() {
+    fabric.Image.fromURL('images/map.gif', img => {
```

Comment:
extracting these strings is gr8 too, saves your eyes from scanning / understanding the entire file when you just want to change the path to the image